### ISSUE TITLE
Cannot chose destination

### ISSUE THREAD

**Issue by shamil** [opendistro-for-elasticsearch/alerting-kibana-plugin#88]
I've many destinations about 100 of them. 
Sadly I can't chose destination over first 20 in the `Configure Actions` section.

I'm running version `0.10.0.0`, Is this known bug?

**Comment by shamil** [comment]
It seems like the request to get destination setting the `searchsze` to `200`, but the actual result is only `20`

if I do `curl` like this:

```
curl -s 'http://localhost:5601/api/alerting/destinations?searchsize=200' | jq '.destinations | length'
```

I get `20`

**Comment by mihirsoni** [comment]
Hi @shamil  Yes there was bug and we fixed it in other version. We backport mostly security fix and any major breaking change. 

Is it hard for you to upgrade to newer version ?

**Comment by shamil** [comment]
@mihirsoni unfortunately upgrading form 6.x to 7.x is a big effort for our clusters and services. We will do it sooner or later, but this will take time

**Comment by ftianli-amzn** [comment]
This bug was fixed in ODFE 1.2.0, which is compatible with ES 7.2.0, and versions above. Currently we have no plan to backport it.
